petastorm/etl/__init__.py

```diff
@@ -31,8 +31,8 @@ def index_name(self):
         return None
 
     @abc.abstractproperty
-    def column_names(self):
-        """ Return list of column(s) reuired to build index."""
+    def field_names(self):
```

Comment:
Thinking about this more this actually doesnt really break pickling, since method/property name changes do not matter. However this is still not a good idea to change. What will happen is implementations of this class will need to update (which is a breaking change) and we need to be sure not to update member variables (so changing `_column_name` to _field_name` in the `SingleFieldIndexer` was actually the breaking change in terms of pickling. That didnt break my tests since the `_field_name` is only used when creating the indexes but still this is all dangerous. I will add a comment that the object is pickled and fields should not be changed.